### ISSUE TITLE
UI fails to expand / collapse node in warc after a leaf node is clicked

### ISSUE THREAD

**Issue by sparkhi** [digital-preservation/droid#359]
Observed on Droid v6.4
Steps to reproduce 
- Run Droid 
- Create a new profile 
- Add a warc file 
- Click start to start profiling 
- Expand a node 
- Click on the leaf node to select it 
---- At this point an NPE is thrown `at uk.gov.nationalarchives.droid.gui.treemodel.DefaultCellRenderer.getFilterStatus(DefaultCellRenderer.java:150)`
- Now click on the parent of the selected node to collapse it 
---- The node collapses but chevron is not updated 
---- From this point, the expand / collapse does not work reliably

**Comment by sparkhi** [comment]
I've investigated and found out that the reason behind the NPE and consequently the unstable behaviour is due to accessibility provider. My default java 9 installation has an `accessibility.properties` file listing a provider, if the provider is missing, it could be the cause of this error.

I've tried with other versions of JAVA (11 and 12) and also the same version (8) of java coming from SDKMAN instead of my default installation and they all work okay.

I then commented out the provider line in `accessibility.properties` and it worked for the default java installation as well.

I think, we can defer this bug as it is likely to be related to my system rather than the packaged bundle of DROID.
